parkings/api/operator/parking.py

```diff
@@ -101,18 +104,22 @@ class OperatorAPIParkingViewSet(mixins.CreateModelMixin, mixins.UpdateModelMixin
     serializer_class = OperatorAPIParkingSerializer
 
     def _get_zone(self):
+        if self.request.data.get('domain'):
+            domain = EnforcementDomain.objects.get(code=self.request.data.get('domain'))
+        else:
+            domain = EnforcementDomain.get_default_domain()
```

Comment:
We can avoid a database query here by not looking up the EnforcementDomain object from the database. Let's rather just take the domain_code to a variable as a string and then use that in the query below, like: `PaymentZone.objects.get(domain__code=domain_code, code=zone_code)`.

But that optimization change can be done in another PR to get this merged already.

Reply:
I ended up rewriting this with SlugRelatedField, since it seemed to make the code cleaner: https://github.com/City-of-Helsinki/parkkihubi/pull/151